assets/src/edit-story/components/panels/animation/directionRadioInput.js

```diff
@@ -21,43 +21,49 @@ import styled, { css } from 'styled-components';
 /**
  * WordPress dependencies
  */
-import { __, sprintf } from '@wordpress/i18n';
+import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import { DIRECTION } from '../../../../animation';
+import { DIRECTION, ROTATION } from '../../../../animation';
 
 const Svg = styled.svg`
   display: block;
-  height: 10px;
-  width: 10px;
+  height: ${({ size }) => size};
+  width: ${({ size }) => size};
   fill: none;
   transform-origin: 50% 50%;
 `;
 
 const Icon = styled.div`
   padding: 4px;
-  transform: rotate(
-    ${({ direction }) => {
-      switch (direction) {
-        case DIRECTION.RIGHT_TO_LEFT:
-          return '270deg';
-        case DIRECTION.TOP_TO_BOTTOM:
-          return '180deg';
-        case DIRECTION.LEFT_TO_RIGHT:
-          return '90deg';
-        default:
-          return '0deg';
-      }
-    }}
-  );
+  transform: ${({ direction }) => {
+    switch (direction) {
+      case DIRECTION.RIGHT_TO_LEFT:
+        return 'rotate(270deg)';
+      case DIRECTION.TOP_TO_BOTTOM:
+        return 'rotate(180deg)';
+      case DIRECTION.LEFT_TO_RIGHT:
+        return 'rotate(90deg)';
+      case ROTATION.COUNTER_CLOCKWISE:
+        return 'rotateX(180deg) rotateZ(90deg)';
+      default:
+        return 'rotate(0deg)';
+    }
+  }};
 `;
 
 const Direction = ({ className, direction }) => (
   <Icon className={className} direction={direction}>
-    <Svg viewBox="0 0 10 11">
-      <path d="M5 11L5 1M5 1L9 5M5 1L1 5" />
-    </Svg>
+    {Object.values(DIRECTION).includes(direction) ? (
+      <Svg size="10px" viewBox="0 0 10 11">
+        <path d="M5 11L5 1M5 1L9 5M5 1L1 5" />
+      </Svg>
```

Comment:
For legibility, it might be nice to store these SVGs as variables so that if one needs to be replaced or altered ever it's easier to tell the purpose behind the two direction/rotation icons just by reading through code.